suite/tests/client-interface/drreg-test.c

```diff
@@ -431,6 +543,77 @@ main(int argc, const char *argv[])
         test_asm_faultL();
     }
 
+    /* For some aflags restore tests below we do not use SIGILL to raise the
+     * fault. This is because the undefined instr on AArchXX is assumed to
+     * read aflags, and therefore restores aflags automatically. So the
```

Comment:
Hmm but this is because the decoder is incomplete?  Seems worth mentioning in the issue on that so ARM is aware, and xref the issue number here.

Reply:
Done. Added the comment on i#4849 too.